src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidatorTest.java

```diff
@@ -71,6 +71,8 @@ class OverseasEntitySubmissionDtoValidatorTest {
     private OwnersAndOfficersDataBlockValidator ownersAndOfficersDataBlockValidator;
     @Mock
     private TrustDetailsValidator trustDetailsValidator;
+    @Mock
+    private HistoricalBeneficialOwnerValidator historicalBeneficialOwnerValidator;
```

Comment:
the mock isn't being referenced in the test, you should verify that it is being called in the expected places

Reply:
thanks @markpit I have corrected it.